src/Helper/EditorHelper.php

```diff
@@ -46,18 +46,13 @@ public function fromString($string)
             throw new \RuntimeException('No EDITOR environment variable set.');
         }
 
-        if (defined('PHP_WINDOWS_VERSION_MAJOR')) {
```

Comment:
I give-up.... 😑 This is just not working. You can't do this in Docker.
Unless anyone is wiling to work on this, or a confirmed working idea.